app/jobs/session_trim_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class SchoolDataImporterJob < CronJob
```

Comment:
That name doesn't sound right ;)

Reply:
Also, is it actually triggered? The school import job, which this is clearly copied from, is not actually triggered at any point, because we don't want to worry about school updates yet. But this sounds like something we would like to run.

Reply:
Yeah, that was something I was actually about to ask you (and clearly forgot) - how are our Cron jobs scheduled? I can clearly see we have cron pattern defined here so I think i assumed this will be scheduled automatically on deploy with eager-load.